zScanner/Coordinators/NewDocumentCoordinator.swift

```diff
@@ -136,19 +136,20 @@ class NewDocumentCoordinator: Coordinator {
         pop()
     }
     
-    private func savePagesToDocument(_ pages: [UIImage]) {
+    private func savePagesToDocument(_ pages: [Picture]) {
 
         // Store images
         pages
             .enumerated()
             .forEach({ (index, image) in
-                let page = PageDomainModel(image: image, index: index, correlationId: newDocument.id)
+                let page = PageDomainModel(picture: image, index: index, correlationId: newDocument.id)
                 newDocument.pages.append(page)
             })
         }
     
     private static func steps(for mode: DocumentMode) -> [Step] {
-            return [.folder, .photos, .documentType]
+//            return [.folder, .photos, .documentType]
```

Comment:
Don't commit testing changes

Reply:
Sorry, my bad.

Reply:
It's not resolved, I can still see it 👀

Reply:
I like the TODO! It's just too vague description. I guarantee you that you will not now what "it will be done" means in couple of months. I've put so many todos like this myself and I was wondering later what the heck I meant by that. Please but more specific description on where the todo can be fixed/removed and how (what code to remove - in this case ucomment and delete the temporary code). 

- just be more specific, please. It's a good habit.

Reply:
It makes sense, thank you for your advice.